docs/pages/operator/init-manifests.mdx

```diff
@@ -17,4 +18,56 @@ init:
     apiVersion: v1
     kind: ConfigMap
     ...
-```
\ No newline at end of file
+```
+
+## Applying charts on vcluster initialization
+vcluster now supports applying helm charts while initializing a new vcluster. Currently 2 methods of applying charts are supported:
+1. Upstream Mode
+2. Bundle Mode
+
+#### Upstream Mode
```

Comment:
Please modify this to 3 #s instead of 4, for the docusaurus autogenerating.
#### Upstream Mode
to
### Upstream Mode